File: Backend/src/main/java/edu/icet/dto/Hospital.java
package edu.icet.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Hospital {
    private int hospitalId;
    private String name;
    private String location;
    private String phone;
    private String email;
    private String description;
}

File: Frontend/src/app/model/hospital.ts
export class Hospital {
    hospitalId: number;
    name: string;
    location: string;
    phone: string;
    email: string;
    description: string;
  
    constructor(
      hospitalId: number,
      name: string,
      location: string,
      phone: string,
      email: string,
      description: string
    ) {
      this.hospitalId = hospitalId;
      this.name = name;
      this.location = location;
      this.phone = phone;
      this.email = email;
      this.description = description;
    }
  }
  